src/client/Microsoft.Identity.Client/Microsoft.Identity.Client.csproj

```diff
@@ -21,7 +21,23 @@
   <PropertyGroup Condition="'$(TargetFramework)' == '$(TargetFrameworkNetStandard)'">
     <DefineConstants>$(DefineConstants);NETSTANDARD_RUNTIME</DefineConstants>
   </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(TargetFramework)|$(Platform)'=='Debug|net45|AnyCPU'">
```

Comment:
>Debug|net45|AnyCPU' [](start = 80, length = 19)

really?

Reply:
@trwalke - please add this MSBuild property in the existing property group for .NET45 which can be found here: https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/blob/a013e8c10b7c10bbfac84e318000d591f3a35373/src/client/Directory.Build.props#L87